test/workflow-status/testpipeline.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const { task, join } = require('../../..')
+
+const fs = require('fs')
+const through = require('through2')
+const { execFile } = require('child_process')
+
+// const myTask = task({
```

Comment:
cleanup this pipeline (remove comments, call `anotherTask` `aTask` or something more reasonable since there will be only one, "another" makes no sense)